examples/nextjs-13/package.json

```diff
@@ -10,8 +10,7 @@
     "lingui:extract": "lingui extract"
   },
   "dependencies": {
-    "@lingui/core": "^4.0.0-next.2",
-    "@lingui/react": "^4.0.0-next.2",
```

Comment:
I am guessing it already comes with `@lingui/react`

Reply:
I think it still needed. 
It works only because of "phantom dependency". Means some other dependency declaring this and because they all hoisted to the top node_modules - it works. But this is a bad pattern.

If you have explicit import like `import {i18n} from "@lingui/core"` you should explicitly state it in dependencies. 

More on phantom dependencies topic you can read here: https://rushjs.io/pages/advanced/phantom_deps/

Reply:
Will do ✅

Reply:
I am ready to push the change to re-add core back but just to be sure: adding lingui core does not impact `yarn.lock`, only `package.json` 🙄.

Do you want me to push it?

Reply:
yes